docs/reference/aggregations/bucket/frequent-items-aggregation.asciidoc

```diff
@@ -49,14 +49,26 @@ A `frequent_items` aggregation looks like this in isolation:
 |`size` | (integer) The number of top item sets to return. | Optional | `10`
 |===
 
+
+[discrete]
+[[frequent-items-fields]]
+==== Fields
+
+Supported field types for the analyzed fields are keyword, numeric, ip, date, 
+and array. You can also add runtime fields to your analyzed fields.
+
+If the combined cardinality of the analyzed fields are high, then the 
+aggregation might require a significant amount of system resources.
+
 [discrete]
 [[frequent-items-minimum-set-size]]
 ==== Minimum set size
 
 The minimum set size is the minimum number of items the set needs to contain. A 
 value of 1 returns the frequency of single items. The higher the minimum set 
-size the less items are returned. For example, the item set `orange, banana, 
-apple` is only returned if the minimum set size is 3 or lower.
+size the less items are returned. For example, the item 
```

Comment:
"the less" sounds not appropriate, because the agg returns max `size` item sets. I suggest:

Only item sets that contain at least `minimum_set_size` items are returned. For example...